### DISCUSSION TITLE
Lowering output power on TX - switchable with arming switch

### DISCUSSION THREAD

**Discussion by kakemanx** [ExpressLRS/ExpressLRS#1530] [Ideas]
A script (or something) that will set the expressLRS to 25mw (or lower output power,maybe adjustable?)) as soon as You disarm, and then again raising the output power on arming..
First it will save battery, second it might prevent overheating/frying your module if you forget the antenna ... obviously i just did this :rolling_eyes::sweat_smile:
Had the module for a week before i managed to do this :person_shrugging:

I mean, it should be possible, and for the obvious disadvantage of disarming at a distance causing a failsafe caused by lower output power either a delay for a few seconds before lowering the power, or just knowing it could happen would still be prefered over frying you module :)

Also an option to run the TX on lower than 25mw for benchtesting purposes, again for saving battery etc.

**Comment by avsaase** [comment]
Have you tried the dynamic power feature?

**Comment by kakemanx** [comment]
No, because they Report it to be unreliable, so i never did 🤷🙄

fre. 15. apr. 2022, 11:43 skrev Alexander van Saase <
***@***.***>:

**Comment by kakemanx** [comment]
But its a good point.. I should...
How far down does it lower the output power in dynamic mode?

Also, i love erls, thanks to all devs for their hard work 🤗

fre. 15. apr. 2022, 11:43 skrev Alexander van Saase <
***@***.***>:

**Comment by SunjunKim** [comment]
With dynamic power the output power lowered all the way to the minimum when your craft is close.
You can disable dynamic power on aux9-12 ch, and link your arming switch to that ch will give you the desired feature
